src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -281,4 +304,44 @@ private function varToString($var): string
 
         return (string) $var;
     }
+
+    private function parseControllerDefinition(callable $controller): ?array
+    {
+        if (is_string($controller) && false !== strpos($controller, '::')) {
+            $controller = explode('::', $controller);
+        }
+
+        if (is_array($controller)) {
+            try {
+                $r = new \ReflectionMethod($controller[0], $controller[1]);
+
+                return array(
+                    'file' => $r->getFileName(),
+                    'line' => $r->getStartLine(),
+                );
+            } catch (\ReflectionException $e) {
+                return null;
```

Comment:
try `new \ReflectionClass($controller[0])`?

Reply:
To get only the file?

Reply:
Yes, WDYT ?

Reply:
I don't understand. Sorry.
If the method does not exist in the first place Symfony throw an error.
I don't want to add dead code to Symfony, so please give me a reproducer ;)